File: Python/PerformanceTest.py
import time

total_duration = 0
num_runs = 100

for i in range(num_runs):
    start_time = time.time()

    # Your code to be tested goes here
    for i in range(1000000):
        j = i + 1

    end_time = time.time()
    duration = (end_time - start_time) * 1000 # Time in milliseconds
    total_duration += duration

average_duration = total_duration / num_runs
print("Average execution time over {} runs: {:.2f} ms".format(num_runs, average_duration))

File: Java/PerformanceTest.java

public class PerformanceTest {
    public PerformanceTest() {
    }

    public static void main(String[] var0) {
        long totalDuration = 0L;
        short numRuns = 1000;

        for(int i = 0; i < numRuns; ++i) {
            long start = System.nanoTime();

            for(int j = 0; j < 1000000; ++j) {
                int k = j + 1;
            }

            long end = System.nanoTime();
            long duration = (end - start) / 1000000L; // Time in milliseconds
            totalDuration += duration;
        }

        double averageDuration = (double) totalDuration / numRuns;
        System.out.printf("Average execution time over %d runs: %.2f ms", numRuns, averageDuration);
    }
}

